packages-web/maps/src/components/MapsContainer.ts

```diff
@@ -220,20 +222,23 @@ class MapsContainer extends Component<MapsContainerProps, MapsContainerState> {
         );
     }
 
-    private onClickMarker = (event: LeafletEvent & google.maps.MouseEvent, locationAttr: DataSourceLocationProps) => {
+    private onClickMarker = (
+        event: LeafletEvent & GoogleMapsClickEvent,
+        locationAttr: DataSourceLocationProps
+    ): void => {
         const { locations } = this.state;
-        const latitude = this.props.mapProvider === "googleMaps" ? event.latLng.lat() : event.target.getLatLng().lat;
-        const longitude = this.props.mapProvider === "googleMaps" ? event.latLng.lng() : event.target.getLatLng().lng;
-        this.executeAction(
-            locations[
-                locations.findIndex(targetLoc => targetLoc.latitude === latitude && targetLoc.longitude === longitude)
-            ],
-            locationAttr
-        );
+        const Guid = this.props.mapProvider === "googleMaps" ? event.options.Guid : event.target.options.Guid;
+
+        if (Guid) {
```

Comment:
Static marker should be clickable to... they don't receive not context. (I assume)

Reply:
What should we do when we receive an object without GUID?

Reply:
The current implementation does not allow to click on static markers because if expects to send the entity to the microflow/nanoflow/page. There are few checks in `MapsContainer.ts`, `GoogleMaps.ts` and `LeafletMap.ts` to dont execute the action if there is no mxobject